swe.m

```diff
@@ -53,10 +53,10 @@
         %==================================================================
     case 'asciiwelcome'                          %-ASCII swe banner welcome
         %==================================================================
-        disp( '   ___          ___           _      ___      ___        ');
-        disp( '  / __) _    _ | __)   _  _  /_|    (__ \    | _ |       ');
-        disp( '  \__ \ \\/\// | __)   \\//   || _  / __/ _  | _ |       ');
-        disp( '  (___/  \/\/  |___)    \/    |||_| \___)|_|   |_|       ');
+        disp( '   ___          ___           _      ___     _  _        ');
```

Comment:
@TomMaullin - Ack!  Didn't realise that this ASCII art was manually crafted!

Can we **not** put the version # into the ascii art?  That's just going give continuing headaches.